README.md

```diff
@@ -14,20 +14,21 @@ To get started, check out [contributing](docs/CONTRIBUTING.md).
 ## Packages
 
 <!-- prettier-ignore-start -->
-| Package | Version | Description |
-| --- | --- | --- |
-| [@littlespoon/alert](packages/alert) | [![NPM version](https://img.shields.io/npm/v/@littlespoon/alert.svg)](https://www.npmjs.com/package/@littlespoon/alert) | Alert |
-| [@littlespoon/box](packages/box) | [![NPM version](https://img.shields.io/npm/v/@littlespoon/box.svg)](https://www.npmjs.com/package/@littlespoon/box) | Box |
-| [@littlespoon/breadcrumbs](packages/breadcrumbs) | [![NPM version](https://img.shields.io/npm/v/@littlespoon/breadcrumbs.svg)](https://www.npmjs.com/package/@littlespoon/breadcrumbs) | Breadcrumbs |
-| [@littlespoon/button](packages/button) | [![NPM version](https://img.shields.io/npm/v/@littlespoon/button.svg)](https://www.npmjs.com/package/@littlespoon/button) | Button |
-| [@littlespoon/checkbox](packages/checkbox) | [![NPM version](https://img.shields.io/npm/v/@littlespoon/checkbox.svg)](https://www.npmjs.com/package/@littlespoon/checkbox) | Checkbox |
-| [@littlespoon/divider](packages/divider) | [![NPM version](https://img.shields.io/npm/v/@littlespoon/divider.svg)](https://www.npmjs.com/package/@littlespoon/divider) | Divider |
-| [@littlespoon/icons](packages/icons) | [![NPM version](https://img.shields.io/npm/v/@littlespoon/icons.svg)](https://www.npmjs.com/package/@littlespoon/icons) | Icons |
-| [@littlespoon/label](packages/label) | [![NPM version](https://img.shields.io/npm/v/@littlespoon/label.svg)](https://www.npmjs.com/package/@littlespoon/label) | Label |
-| [@littlespoon/link](packages/link) | [![NPM version](https://img.shields.io/npm/v/@littlespoon/link.svg)](https://www.npmjs.com/package/@littlespoon/link) | Link |
-| [@littlespoon/tag](packages/tag) | [![NPM version](https://img.shields.io/npm/v/@littlespoon/tag.svg)](https://www.npmjs.com/package/@littlespoon/tag) | Tag |
-| [@littlespoon/theme](packages/theme) | [![NPM version](https://img.shields.io/npm/v/@littlespoon/theme.svg)](https://www.npmjs.com/package/@littlespoon/theme) | Theme (colors, fonts) |
-| [@littlespoon/typography](packages/typography) | [![NPM version](https://img.shields.io/npm/v/@littlespoon/typography.svg)](https://www.npmjs.com/package/@littlespoon/typography) | Typography |
-| [@littlespoon/ui](packages/ui) | [![NPM version](https://img.shields.io/npm/v/@littlespoon/ui.svg)](https://www.npmjs.com/package/@littlespoon/ui) | UI (collection of modules) |
-| [storybook](packages/storybook) | Private | [Storybook](https://littlespoon.netlify.app/) |
+| Package                                          | Version                                                                                                                             | Description                                   |
+|--------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------|
```

Comment:
Nit: Do you mind reverting the whitespace changes? I tried to make prettier ignore this table

Reply:
Reverted in 5ad5b7cce85156a538928136ae7150b666546870 and 955b12f7d0569e08f9cbfcf6151043f6db61e99b